README.md

```diff
@@ -17,25 +17,28 @@ https://discord.gg/5JmwPJf
 - OpenGL (Version >= 3.3)
 
 #### Building
+Building of this project is a simple endeavour, requiring only a few commands.
 
-- Either clone https://github.com/GentenStudios/quartz-engine or fork and then clone your fork. You will have to either clone with the
-``--recursive`` option, or you can run: ``git submodule update --init`` after cloning.
+Initially, a [clone](https://help.github.com/en/articles/cloning-a-repository) of the repository will be required, however, if you plan on making edits first, make sure you [fork](https://help.github.com/en/articles/fork-a-repo) the repo beforehand, and clone that.
 
-- Run `cmake -H. -Bbuild` inside the root directory
+An example command for cloning the repository is:
 
-- Then either build with `cmake --build build` or follow the instructions for your compiler
+<dl>
+git clone https://github.com/GentenStudios/quartz-engine.git
+</dl>
 
-  ##### Visual Studio
+Once cloned, enter the directory, by doing `cd quartz-engine``, and complete the following commands:
 
-  - Right click on `quartz-sandbox` and click `Set as StartUp project`
-  - Click `Local Windows Debugger` to run
+<dl>
+cmake -H. -BBuild
+cmake --build Build
+</dl>
 
-  ##### Make
-
-  - run `cd build` and then `make`.
-  - Run the resulting binary
+These commands will take a fair amount of time to complete, the first one configures the project and appropriate compilers, whereas the second command actually builds the project using those configurations. To run the executable, for MSVC/Windows users, go into Build/QuartzSandbox and move QuartzSandbox.exe out of Debug and into this folder. You can then run it.
```

Comment:
Explaining what is happening is not necessary, the disclaimer about time may be okay to include but the rest is non-necessary fluff.

Reply:
Too descriptive. Would change to `Move to the root directory with cd build, and then compile the source with make,` or something along the lines of this